### ISSUE TITLE
Some grey (in light theme, might be other color in other themes) rectangle remains behind active tabs

### ISSUE THREAD

**Issue by Nyekomimi** [wilfredwee/photon-australis#11]
This make entire point of skin moot (since that rectangle makes it look squarish ect) and keeps "tabs" part of hud too high (as it is in vanilla Firefox Quantum), rather than making it shorter (like it should be and like it was in previous Firefox instances)

**Comment by wwsvs** [comment]
Yes, I have the exact same problem.  I'm on Win7 64bit Ultimate.   I tried going with the dark theme and I still have the same issues and the dark theme does not change the area above and where the tabs would appear.   Why did FF change this?  It is really ugly now and frankly no faster than before.

**Comment by wilfredwee** [comment]
In Win10 and OSX they are properly curved on hover + when selected. Transparent otherwise.

Unfortunately I don't have a machine running Windows 7 anywhere to test this :(

I'll keep this thread open in case someone may create a pull request to fix this bug? Screenshots etc welcomed.

**Comment by peekaa** [comment]
When using fe. this https://addons.mozilla.org/cs/firefox/addon/summer-time-converse-madonna/, the tabs/tabs backgrounds are ok. 
![obrazek](https://user-images.githubusercontent.com/3026590/32909826-e647d54a-cb07-11e7-8259-3842d0b0a300.png)

**Comment by teijiIshida** [comment]
I can confirm this bug in my Windows 7 x64. Also, it only bugs with the Firefox's default, light, and dark theme. If you use other themes, it doesn't have the weird background bug.

**Comment by doej1367** [comment]
Windows 7 Pro x64 - same here

**Comment by Pinefall** [comment]
There seems to be a single pixel line to the right of the opaque block that is somewhat transparent https://i.imgur.com/haU8cC8.png

**Comment by Pinefall** [comment]
it seems that the rounded tabs themselves work there is just an opaque block behind everything https://i.imgur.com/QY0wTCW.png

**Comment by slepmog** [comment]
This appears to be Firefox issue to begin with: https://bugzilla.mozilla.org/show_bug.cgi?id=1418796, which was closed as a duplicate of https://bugzilla.mozilla.org/show_bug.cgi?id=1404136.

**Comment by e4lam** [comment]
I'm hitting this bug too. Is there some setting in the userChrome.css, where as a workaround, I can tell Firefox to use an opaque window title bar instead?

**Comment by wilfredwee** [comment]
If anyone is still interested, you can try adding this CSS:

```
.tabbrowser-tab {
  background-color: transparent !important;
}

```

**Comment by peekaa** [comment]
Yes, confirm, works with default layout well in w7. Thank you. 
![screenshot_3](https://user-images.githubusercontent.com/3026590/34083916-466f497a-e378-11e7-8c54-dcfed9826b3d.jpg)
